app/src/main/java/uc/edu/klopfsea/groupscheduler/ui/main/CustomCalendar.kt

```diff
@@ -2,5 +2,10 @@ package uc.edu.klopfsea.groupscheduler.ui.main
 
 class CustomCalendar {
 
+    List<EventDay> events = new ArrayList<>();
+    Calendar calendar = Calendar.getInstance();
+    events.add(new EventDay(calendar, R.drawable.sample_icon));
+    CalendarView calendarView = (CalendarView) findViewById(R.id.calendarView);
+    calendarView.setEvents(events);
```

Comment:
Just a little something I wrote up, maybe a way for the calendar to list out the days people have scheduled.

Reply:
Not at my main computer right now so I can't look into this fully but it appears in some way this change broke the project. CricleCI says it no longer compiles and this is where it fails.